libs/core/src/lib/carousel/carousel.component.html

```diff
@@ -63,6 +63,7 @@
             <ng-container *ngIf="pageIndicator">
                 <ng-container *ngFor="let item of pageIndicatorsCountArray; let i = index">
                     <li
+                        [attr.aria-label]="_slidesCopy[i].title"
```

Comment:
Maybe add it as an input so the user can provide their own label and in case the user does not provide such you can generate the one that you have now.